### PR TITLE
xAPISaverPlugin added

### PR DESCRIPTION
**Pull request by dpeite**

### CODE REVIEW

**Comment by rubenrua** [comment]
@brapifra Could you review this when @dpeite add the last commit? Thank you

File: plugins/es.teltek.paella.usertracking.xAPISaverPlugin/xAPISaverPlugin.js [hunk 1]
```diff
@@ -0,0 +1,405 @@
...
+				// myparams['verb'] = 'https://w3id.org/xapi/video/verbs/paused';
+				// myparams['limit']	= 1;
+				// var ret = ADL.XAPIWrapper.getStatements(myparams);
+				// console.log(ret)
+
+			})
+			paella.events.bind(paella.events.timeUpdate, function(event,params){
+				self.current_time.push(params.currentTime)
+				self.progress = self.get_progress(params.currentTime, params.duration)
+				if (self.progress == 1 && self.completed == false){
```
Comment (brapifra) [inline plugins/es.teltek.paella.usertracking.xAPISaverPlugin/xAPISaverPlugin.js]: Strict Equality Comparison (===) should be used

File: plugins/es.teltek.paella.usertracking.xAPISaverPlugin/xAPISaverPlugin.js [hunk 2]
```diff
@@ -0,0 +1,405 @@
...
+			this.speed = null
+			this.quality = null
+			this.fullscreen = false
+			this.title = ""
+			this.user_agent = ""
+			this.total_time = 0
+			this.total_time_start = 0
+			this.total_time_end = 0
+
+			var self = this
```
Comment (brapifra) [inline plugins/es.teltek.paella.usertracking.xAPISaverPlugin/xAPISaverPlugin.js]: Use const/let

**Review by brapifra — CHANGES_REQUESTED** [review]

**Comment by rubenrua** [comment]
Please open a pull request upstream (polimediaupv/paella).

**Pull request closed without merging**